### PR TITLE
Nomenclature

### PR DESCRIPTION
**Pull request by OctopusBob**
Added a terminology page to the getting started section, included a link in the getting started guide to the new page.

https://preprod.octopus.com/docs/getting-started/terminology

### CODE REVIEW

File: docs/getting-started/terminology.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Terminology
+description: A page containing the definitions of terms used in Octopus Deploy.
+position: 10
+---
+
+As you start using Octopus Deploy, you are going to learn a lot of new terms.  Below is a list of common terms used through the product and documentation.
+
+- **Octopus Server**: the service responsible for hosting the Octopus UI and orchestrating deployments.
```
Comment (wordlee) [inline docs/getting-started/terminology.md:9]:
```suggestion
- **Octopus Server**: the service responsible for hosting the Octopus Web Portal, Rest API, and orchestrating deployments.
```

File: docs/getting-started/index.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ Before you can deploy software with Octopus Deploy, you need to bundle all the f
 
 The guide below provides more information on how to [package your software](#package-software).
 
+## Terminology
+
+As start using Octopus Deploy, you are going to learn a lot of new terms.  So much so it can feel overwhelming.  You can view a list of common terms on the [terminology page](docs/getting-started/terminology.md).
```
Comment (wordlee) [inline docs/getting-started/index.md:40]: Slight tweak, also I removed overwhelm so as not to pre-overwhelm readers :)
```suggestion
As you start using Octopus Deploy, you will learn a lot of new terms. You can view a list of common terms on the [terminology page](docs/getting-started/terminology.md).
```

File: docs/getting-started/terminology.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- **Self-Hosted**: when you install the Octopus Server on your infrastructure.
+- [**Octopus Cloud**](/docs/octopus-cloud/index.md): the hosted version of Octopus Server; but you still install tentacles on your infrastructure that connect to Octopus Cloud.
+- **Task**: a unit of work performed by the Octopus Server.  A task can be a deployment, a machine health check, a runbook run, and more.  All tasks are dropped onto the task queue and picked up in a FIFO order (unless the task is scheduled to run at a specific time).
+- [**Task Cap**](/docs/support/increase-the-octopus-server-task-cap.md): how many concurrent tasks the Octopus Server can process.  For self-hosted instances this can be increased from the default of 5.
+- **Instance**: the database, file share, and 1 to N nodes running the Octopus Server service.  Each self-hosted Octopus Deploy license allows for three active instances.
+- [**Node**](/docs/administration/high-availability/managing-high-availability-nodes.md): an individual server running the Octopus Server in an Octopus Instance.  
+- [**Infrastructure**](/docs/infrastructure/index.md): what the Octopus Server will deploy to.
```
Comment (wordlee) [inline docs/getting-started/terminology.md:16]:
```suggestion
## Infrastructure

 Your [**Infrastructure**](/docs/infrastructure/index.md) is servers, services, and accounts where the Octopus Server will deploy your software.
```

File: docs/getting-started/terminology.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    - [**Health Check**](/docs/infrastructure/deployment-targets/machine-policies.md#health-check): a task Octopus periodically runs on deployment targets and workers to ensure that they are available.
+    - [**Machine Policies**](/docs/infrastructure/deployment-targets/machine-policies.md): groups of settings that can be applied to Tentacle and SSH endpoints used for health checks, updating calamari, and more.
+    - [**Machine Proxies**](/docs/infrastructure/deployment-targets/proxy-support.md): allow you to specify a proxy server for Octopus to use when communicating with a Tentacle or SSH Target; you can also specify a proxy server when a Tentacle and the Octopus Server make web requests to other servers.
+    - [**Accounts**](/docs/infrastructure/deployment-targets/index.md#accounts): credential details used during your deployments, including things like username/password, tokens, Azure and AWS credentials, and SSH key pairs.  
+- [**Projects**](/docs/projects/index.md): where the deployment process, configuration variables, runbooks are defined and configured to deploy and manage your software.
```
Comment (wordlee) [inline docs/getting-started/terminology.md:28]: Slight change to the wording to make it work as introductory text.
```suggestion
## Projects

[**Projects**](/docs/projects/index.md) contain the deployment process, configuration variables, and runbooks to deploy and manage your software.
```

File: docs/getting-started/terminology.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    - [**Release**](/docs/releases/index.md): a snapshot of the deployment process and the associated assets (packages, scripts, variables) as they existed when the release was created. 
+    - [**Variables**](/docs/projects/variables/index.md): a value stored in the Octopus Server for use in different phases of your deployments.  Variables can be scoped to environments, steps, and more.  Variables allow you to have a consistent deployment process across your infrastructure without having to hard-code or manually update configuration settings that differ across environments, deployment targets, channels, or tenants.
+    - [**Manual Interventions**](/docs/deployment-process/steps/manual-intervention-and-approvals.md): the approval step in Octopus Deploy.  Manual interventions can be scoped to specific teams and environments.
+    - [**Runbooks**](/docs/runbooks/index.md): used to automate routine maintenance and emergency operations tasks like infrastructure provisioning, database management, and website failover and restoration.    
+- **Library** - stores items used across multiple projects
```
Comment (wordlee) [inline docs/getting-started/terminology.md:35]: I've suggested artifacts and assets, does artifacts work in this context?
```suggestion
## Library 

The Octopus library stores artifacts and assets that can be used across multiple projects
```

**Comment by wordlee** [comment]
I made a couple of suggestions to break the list up a bit and make it less daunting (just converted top level list items to headings). I'd still like to go through and review the language on each point. Will do that as soon as I can.

File: docs/getting-started/terminology.md (hunk 5 above)
Comment (OctopusBob) [reply docs/getting-started/terminology.md:35]: I'll update.  We use the term artifact as well (which I will add) in deployments.

The updated line will be:
> The Octopus library is where you store build artifacts and other assets that can be used across multiple projects.

**Comment by OctopusBob** [comment]
Added a new deployment section to hold terms such as task log, artifacts, raw log, change history and more.  Added a few more terms to the list that I can't believe I missed (HA and spaces to name a few)

**Comment by wordlee** [comment]
I pushed some minor line edits to the text. I agree with the intention behind the list (It's a great idea), but I'm a bit worried it might be too extensive and actually increase the potential overwhelm for new users looking to get started. I think we have a couple of options though :)
1. Move the list and place it further down the TOC with the title glossary or common terms.
2. Reduce the list to the minimum items new users need to know to get started.
3. 1 & 2, keep the extended list and keep adding to it, and create a minimal version that lives in the getting started section (use includes so items are automatically updated on both lists).

**Comment by OctopusBob** [comment]
I tried a slightly different approach.  I broke up the pages into various sections.  POC concepts to learn, Pilot phase concepts to learn, and advanced topics.  Does that make it easier to consume?  https://preprod.octopus.com/docs/getting-started/terminology

**Pull request closed without merging**